controlplane/eks/api/v1beta1/awsmanagedcontrolplane_types.go

```diff
@@ -166,6 +167,17 @@ type AWSManagedControlPlaneSpec struct { //nolint: maligned
 	// Amazon VPC CNI addon or if you have specified a secondary CIDR block.
 	// +kubebuilder:default=false
 	DisableVPCCNI bool `json:"disableVPCCNI,omitempty"`
+
+	// VPCCNI defines a list of environment variables to apply to the `aws-node` DaemonSet
+	// +optional
+	VPCCNI VPCCNI `json:"VPCCNI,omitempty"`
```

Comment:
Hmmm, this shouldn't be fully uppercase, right?

Reply:
@Skarlso - do you think it should be `VpcCni` instead?

Reply:
Just to weigh in on this, I think `VPCCNI` makes more sense due to it being an acronym, it also matches the previous stylisation of `DisableVPCCNI`.

Happy to change it however if we think something like `VpcCni` or `VpcCNI` makes more sense

Reply:
I think all options don't sit well....2 acronyms together :vomiting_face: My thoughts on the options:

1. `VPCCNI` - i tend to agree with @charlie-haley, as they are acronyms capitals sort of makes sense and it does fit with the 
`DisableVPCCNI` field. But the `DisableVPCCNI` will be removed in **v1beta2**.
2. `VpcCNI` - i don't like this as it's mixing capitalization for acronyms
3.  `VpcCni`- if there is a strong option against fully caps then this would be my choice.

I don't have a super strong opinion apart from not choosing option 2. @Skarlso @sedefsavas  (or anyone else) any other views?

Reply:
I don't have strong feelings about this so I took the liberty of updating the PR to use `VpcCni`